test/lib/pmd/PmdCatalogWrapper.test.ts

```diff
@@ -97,11 +102,12 @@ describe('PmdCatalogWrapper', () => {
 				before(() => {
 					Sinon.createSandbox();
 					// Spoof a config that claims that only apex is the supported language
-					Sinon.stub(Config.prototype, 'getSupportedLanguages').withArgs(ENGINE.PMD).resolves(['apex']);
+					Sinon.stub(Config.prototype, 'getSupportedLanguages').withArgs(ENGINE.PMD).resolves([LANGUAGE.APEX]);
 					const customJars: Map<string, Set<string>> = new Map();
-					customJars.set('pl/sql', new Set([irrelevantPath]));
-					customJars.set('java', new Set());
+					customJars.set(LANGUAGE.PLSQL, new Set([irrelevantPath]));
```

Comment:
Again, part of the point is to make sure that `pl/sql` gets de-aliased into `plsql`. So leave the string as is.